CommandHelper/LocalPackages/commands/chars/batjoker

```diff
@@ -0,0 +1,27 @@
+costumes:
```

Comment:
This is really good!

The only issue I see is that it the filename up there should be

CommandHelper/LocalPackages/commands/chars/batjoker.yml
not
CommandHelper/LocalPackages/commands/chars/batjoker

notice it should have .yml

thanks!

Reply:
ok im on it